src/main/java/com/openshift/internal/restclient/builders/SourceDeploymentBuilder.java

```diff
@@ -1,88 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
- * All rights reserved. This program is made available under the terms of the
- * Eclipse Public License v1.0 which accompanies this distribution, and is
- * available at http://www.eclipse.org/legal/epl-v10.html
- * 
- * Contributors: Red Hat, Inc.
- ******************************************************************************/
-package com.openshift.internal.restclient.builders;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import com.openshift.internal.restclient.model.BuildConfig;
-import com.openshift.internal.restclient.model.DeploymentConfig;
-import com.openshift.internal.restclient.model.ImageRepository;
-import com.openshift.internal.restclient.model.KubernetesResource;
-import com.openshift.restclient.images.DockerImageURI;
-
-/**
- * SourceDeploymentBuilder supports building OpenShift resources that
- * can be deployed by injecting a user's source into an image.  This builder
- * produces the resources required to build and deploy an image.
- * 
- * @author Jeff Cantrill
- */
-public class SourceDeploymentBuilder {
-
```

Comment:
@adietish I'm not sure I follow your question.  From what I can tell, ImageStream is, at the moment, a 1:1 replacement for ImageRepo